docs/griptape-framework/drivers/embedding-drivers.md

```diff
@@ -177,7 +177,7 @@ from griptape.drivers import (
 from griptape.config import StructureConfig
 
 agent = Agent(
-    tools=[WebScraper(), TaskMemoryClient(off_prompt=False)],
+    tools=[WebScraper(off_prompt=True), TaskMemoryClient(off_prompt=False)],
```

Comment:
for these examples, do we still need `off_prompt=False` for the `TaskMemoryClient` if the default is updated?

Reply:
Not technically, but I figured it couldn't hurt to show examples of being explicit.